src/libcommon/log/sentry/sentryuser.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Infomaniak kDrive - Desktop
+ * Copyright (C) 2023-2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include <string>
+
+namespace KDC {
+
+struct SentryUser {
+    public:
+        SentryUser() = default;
+        SentryUser(const std::string_view &email, const std::string_view &username, const std::string_view &userId)
+            : email(email), username(username), userId(userId), defaultUser(false) {}
+        explicit operator bool() const { return !defaultUser; }
+        inline std::string_view getEmail() const { return email; };
```

Comment:
We usually do not use the prefix `get` for accessors.

Reply:
My bad sorry, here is the correction: [0ba450e](https://github.com/Infomaniak/desktop-kDrive/pull/274/commits/0ba450e39511a39376b5cb2ad06f1da41c44e12b)